doc-Managing_Infrastructure_and_Inventory/topics/Reconfiguring_VMs.adoc

```diff
@@ -1,27 +1,35 @@
 [[Reconfiguring_VMs]]
-= Reconfiguring Virtual Machines (VMware and Red Hat Virtualization Virtual Machines Only)
+= Reconfiguring Virtual Machines
 
 Memory, processors, and disks can be reconfigured on existing VMware and Red Hat Virtualization virtual machines using the *Reconfigure this VM* button.
 
+You can reconfigure multiple components on a virtual machine using one request before confirming the changes with the *Submit* button, or make each reconfiguration separately.
```

Comment:
Hey Dayle,

This sentence has a slight awkwardness to me. I've read it aloud a few times and I feel like you could remove the phrase "before confirming the changes with the *Submit* button, and move it to its own following sentence. Ex: "You can reconfigure multiple components on a virutal machine using one request, or make each reconfiguration separately. Confirm your changes using the with *Submit* button."

Let me know what you think. Other than that everything else in this PR looks ready to go.

Reply:
Hey Chris,
Totally agreed, it does sound awkward in its current state -- I was struggling with the wording here, but thought it was worth pointing out as it's a little annoying to reconfigure each thing individually, in the way we've documented the procedures. Thanks for the wording suggestion! I'll make that change now.